packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -77,7 +77,7 @@ class PatchCommand extends ShorebirdCommand {
         help: '''
 The version of the associated release (e.g. "1.0.0").
 If you are building an xcframework or aar, this number needs to match the host app's release version.
-To target the latest release (e.g. the release that was most recently updated) use --release-version=latest.''',
```

Comment:
Did you confirm on the backend that we're not just checking release.updated_at 😮

Reply:
hmm I thought Bryan changed this behavior but it doesn't look like it 🤷